Sources/Reactive.swift

```diff
@@ -0,0 +1,26 @@
+/// Describes a provider of reactive extensions.
+///
+/// - note: `ReactiveExtensionProvider` does not indicate whether a type is
+///         reactive. It is intended for extensions to types that do not own by
+///         the module so as to avoid name collision and return type ambiguity.
+public protocol ReactiveExtensionProvider: class {}
```

Comment:
Just a little nit: Should it be `ReactiveExtensionsProvider` (i.e. the plural) instead, since there are often more than one extension being provided?